README.md

```diff
@@ -16,16 +16,14 @@ jobs:
     steps:
       - name: Notify release
         uses: nearform/github-action-notify-release@v1
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
 ```
 
 ## Usage
 
 Configure this action in your workflows providing the inputs described below in order to get notified in `x` days after the repo has been updated but no release has happened.
 
 ### `github-token`
-**Required** A GitHub token. See below for additional information.
+**Optional** A GitHub token. See below for additional information.
```

Comment:
let's remove the second part of this sentence because there is no additional information below

Reply:
done